f2/src/BusinessPage.js

```diff
@@ -35,16 +39,28 @@ export const BusinessPage = () => {
                 </P>
               </Stack>
               <BusinessInfoForm
-                onSubmit={(data) => {
+                onSubmit={async (data) => {
+                  if (doneForms && !pages.editOptions) {
+                    pages.nextPage = whatWasAffectedPages.CONFIRMATION
+                  } else {
+                    pages.nextPage = await nextPage(pages)
```

Comment:
Is await necessary here? If you put await in front of a synchronous function, in my impression, the function will return a promise, even though the promise is immediately resolvable. I need to double check this.
If it's not required, I prefer remove it.

Reply:
You're right await is not necessary. I will push an update without await.